src/components/RichEditor/RichEditorBase/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useRef, useState } from "react";
+
+import { EditorState, convertToRaw, RawDraftContentState, convertFromRaw } from "draft-js";
+import { Editor, EditorProps } from "react-draft-wysiwyg";
+
+import BoxBase from "src/components/Boxs/BoxBase";
+
+import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
+
+export interface RichEditorBaseProps extends EditorProps {
+    placeholder?: string;
+    initContent?: string;
+    getValue?: (content: string) => void;
+    height?: number;
+    focus?: boolean;
+}
+
+const RichEditorBase = (props: RichEditorBaseProps) => {
+    const editorRef = useRef<HTMLInputElement>();
+
+    useEffect(() => {
+        if (editorRef.current && props.focus) {
+            editorRef.current.focus();
+        }
+    }, [props.focus]);
```

Comment:
Why do we need this part?

Reply:
I think It need to do some features like focus and show error when invalid input,...